cli/internal/cmd/init.go

```diff
@@ -177,15 +198,18 @@ type initDoer struct {
 	endpoint string
 	req      *initproto.InitRequest
 	resp     *initproto.InitResponse
+	log      debugLog
 }
 
 func (d *initDoer) Do(ctx context.Context) error {
 	conn, err := d.dialer.Dial(ctx, d.endpoint)
+	d.log.Debugf("Dialed init server")
 	if err != nil {
 		return fmt.Errorf("dialing init server: %w", err)
 	}
```

Comment:
```suggestion
	if err != nil {
		d.log.Debugf("Dialing init server failed: %w. Retrying...", err)
		return fmt.Errorf("dialing init server: %w", err)
	}
```
I think in this case its clearer if we only produce a log message if we received an error